src/Products/PloneMeeting/setuphandlers.py

```diff
@@ -149,6 +150,7 @@ def postInstall(context):
         return
     site = context.getSite()
 
+    activate_solr_and_reindex_if_available(site)
```

Comment:
Je me demande s'il ne faudrait pas appeler solr après l'ajout des indexes?  Normalement solr à sa config d'index dans le fichier XML.  Du coup, je me demande même si on ne ferait pas un "if" pour l'ajout des index juste en dessous (addOrUpdateIndexes, addOrUpdateColumns).  Tu pourrais faire activated_solr = activate_solr_and_reindex_if_available(site) et que la méthode ramène True si solr a été activé.  Du coup si "not solr_activated", déclencher les addOrUpdate... sinon c inutile (je crois?).  Pourrais tu tester çà?

Reply:
En fait les indexes sont créés dans Solr via le schema.xml. on peut en créer autrement mais dans ce cas-ci ils sont sensé être déjà dans le schema.

Reply:
Si je skip les addOrUpdateXXX, ça se crache
  File "/srv/cache/eggs/zope.schema-4.2.2-py2.7.egg/zope/schema/_field.py", line 389, in _validate
    raise ConstraintNotSatisfied(value)
ConstraintNotSatisfied: getProposingGroup